README.md

```diff
@@ -11,6 +11,9 @@ This is a Monorepo managed using [Turborepo](https://turbo.build/) and contains
 
 > warning: This package doesn't support AsyncAPI 1.x anymore. We recommend to upgrade to the latest AsyncAPI version using the [AsyncAPI converter](https://github.com/asyncapi/converter-js) (You can refer to [installation guide](/apps/generator//docs//installation-guide.md)). If you need to convert documents on the fly, you may use the [Node.js](https://github.com/asyncapi/converter-js) or [Go](https://github.com/asyncapi/converter-go) converters.
 
+> **Deprecation Notice**: The use of `cli.js` for documentation generation is deprecated and will be removed in future releases. We strongly encourage migrating to the new AsyncAPI CLI. The migration process is straightforward, and you can find the necessary steps in our [migration guide](/apps/generator/docs/deprecate-cli.sj-ag.md). Please ensure that your projects are updated accordingly to avoid any disruptions.
```

Comment:
remember that users do not understand waht `cli.js` is, only maintainers. This notice is for users and they only know that cli is `ag` or `asyncapi-generator`

also please use one of GH supported markdown alerts -> https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts. Probably `warning` is what we need.

and last but no least:
- link `/apps/generator/docs/deprecate-cli.sj-ag.md` transform to a link pointing to `https://asyncapi.com/docs/tools/generator/cli-deprecation` - yes, link is 404 but will not be after we merge this PR. Of course you need to rename`deprecate-cli.sj-ag.md` file to `cli-deprecation.md`
- please provide precise information to this note that removal of functionality will be done in September 2025

Reply:
For the GH supported markdown alerts, do you mean this?
<img width="799" alt="Screenshot 2024-09-16 at 2 26 27 PM" src="https://github.com/user-attachments/assets/2726203b-ba73-4668-95a5-c5d513411566">